Migrate images to server

frandelgado opened the pull request:
This PR adds all of the event's badge images to the /website/public/events/badges/ folder.
all migrated images have their names set to {fancy-id}.{extension}
Note that the event with id 30 does not have a fancy-id so it could not be migrated with a valid name.

In order to update the database to point to the new images, the following sql sentences should be executed:
https://gist.github.com/frandelgado/2ef83f3ef05f2057942eff044eaa939c

The script used to generate the images and the migration file can be found at:
https://gist.github.com/frandelgado/ed14f2ce83cbb935f7cbb8be17355078

frandelgado committed:
Migrate images to server

ragonzal commented:
@frandelgado there ir a committed file `.png`, why?
Did we run any validation comparing: amount of events in API, amounts of `UPDATE` statements, amount of files in folder?

ragonzal reviewed (changes requested):
@frandelgado there ir a committed file `.png`, why?
Did we run any validation comparing: amount of events in API, amounts of `UPDATE` statements, amount of files in folder?

frandelgado committed:
Remove badly formatted image and Add propper image

frandelgado commented:
Right, I left it because even though there was no fancy id for it I still wanted to keep the image in the migration. 
I changed the name to a more sensible one (the one it previously had) and updated the gist to match that name.
